src/modules/fundingManager/oracle/interfaces/ILM_ManualExternalPriceSetter_v1.sol

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity 0.8.23;
+
+/**
+ * @title   Manual External Price Oracle Interface
+ *
+ * @notice  Interface for the manual price feed mechanism that allows setting
+ *          and updating prices for token issuance and redemption operations.
+ *
+ * @dev     This interface extends IOraclePrice_v1 and adds functionality for
+ *          manually setting prices. Both prices (issuance and redemption) must
+ *          be non-zero values and can only be set by the contract owner.
+ *
+ * @custom:security-contact security@inverter.network
+ *                          In case of any concerns or findings, please refer to
+ *                          our Security Policy at security.inverter.network or
+ *                          email us directly!
+ *
+ * @author  Zealynx Security
+ */
+interface ILM_ManualExternalPriceSetter_v1 {
```

Comment:
Not sure if I missed this, but I believe the plan was that this interface would inherit from the IOracle interface. At least that is shown on the Mermaid diagram and you wrote this in the natspec of the interface.

Reply:
I had it added in the contract `LM_ManualExternalPriceSetter_v1`. Moved it here.